content/logs/log_collection/docker.md

```diff
@@ -152,7 +152,7 @@ LABEL "com.datadoghq.ad.instances"='[{"nginx_status_url": "http://%%host%%:%%por
 LABEL "com.datadoghq.ad.logs"='[{"source": "nginx", "service": "webapp"}]'
 ```
 
-Check our [Autodiscovery Guide][8] for more information about Autodiscovery setup and examples.
+Pod Annotation is not yet supported for logs. Check our [Autodiscovery Guide][8] for more information about Autodiscovery setup and examples.
```

Comment:
Not very keen on documenting all the feature that we do not have.

We have this earlier in the page `Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.` 

We could add something like:  `If you are running in Kubernetes and do not have access to container labels, pod annotation will soon be supported.`

Reply:
I'd suggest `do not use container labels` instead of `do not have access to container labels` but otherwise 👍.